src/engraving/libmscore/textlinebase.cpp

```diff
@@ -306,18 +306,22 @@ void TextLineBaseSegment::layout()
         _offset2.setY(0);
     }
 
-    auto alignText = [tl](Text* text) {
+    auto alignBaseLine = [tl](Text* text, PointF& pp1, PointF& pp2) {
+        PointF widthCorrection(0.0, tl->lineWidth() / 2);
```

Comment:
I'm not sure whether this takes into account whether the line is visible or not (`textLineBase()->lineVisible()`)

Reply:
If the line is not visible the function will not be called, as expected, because of [this line](https://github.com/musescore/MuseScore/pull/15428/files#diff-6db3e5aff93698409228e959022f9efb004707d57d265492950b4a7ff5af9da6R447)